### PR TITLE
LPAL-1080 Upgrade service-admin to PHP 8.2

### PR DESCRIPTION
**Pull request by nickdavis2001**
## Purpose

_Upgrade service-admin to PHP _

## Approach

_Update versions in composer.json, have CI use 8.2 version of phpunit and psalm_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by townxelliot — CHANGES_REQUESTED** [review]
You'll also need to modify the github action config so we use php8.2 for unit tests and psalm scan.

File: service-admin/composer.json [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
-        "php": "^8.1",
+        "php": "^8.2",
         "aws/aws-sdk-php": "^3.81",
         "firebase/php-jwt": "^6.0",
         "guzzlehttp/guzzle": "^6.3",
         "laminas/laminas-authentication": "^2.8",
-        "laminas/laminas-cache": "^3.1.2",
-        "laminas/laminas-cache-storage-adapter-memory": "2.0",
+        "laminas/laminas-cache": "^3.9",
+        "laminas/laminas-cache-storage-adapter-memory": "2.2",
```
Comment (townxelliot) [inline service-admin/composer.json:22]: I would make this "^2.2"

**Review by townxelliot — CHANGES_REQUESTED** [review]

File: service-admin/composer.json [hunk 2]
```diff
@@ -13,13 +13,13 @@
         }
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
```
Comment (townxelliot) [inline service-admin/composer.json:16]: I would probably fix this to "8.2". That way we can't accidentally move it to 8.3 without realising.

**Review by townxelliot — CHANGES_REQUESTED** [review]

File: service-admin/composer.json (hunk 1 above)
Comment (nickdavis2001) [reply service-admin/composer.json]: done

File: service-admin/composer.json (hunk 2 above)
Comment (nickdavis2001) [reply service-admin/composer.json]: done

**Comment by townxelliot** [comment]
Because we had to revert the PHP 8.2 migration for service-front, it broke your .github/workflows changes. I think we should probably attempt to push service-admin to PHP 8.2 as it's low risk and shouldn't suffer any performance drops as it gets hardly any load.

**Comment by nickdavis2001** [comment]
@townxelliot I have fixed the conflicts :)

**Review by townxelliot — APPROVED** [review]

**Pull request merged**